src/core/Akka/Actor/ActorCell.Children.cs

```diff
@@ -353,58 +353,37 @@ private bool TryGetChildRestartStatsByName(string name, out ChildRestartStats ch
         /// </summary>
         /// <param name="name">N/A</param>
         /// <returns>N/A</returns>
-        [Obsolete("Use TryGetSingleChild [0.7.1]")]
         public IInternalActorRef GetSingleChild(string name)
```

Comment:
Half the perf optimizations happened here:

* Simplified branching
* Remove `Try... out` stuff since it was all functionally being converted back into `ActorRefs.Nobody` anyway.